main.rb

```diff
@@ -0,0 +1,12 @@
+class Main
+  def read_file
+    if File.exist?("games.log")
+      quakeLog_firstLine = File.open("games.log", &:readline) 
+      return puts quakeLog_firstLine
```

Comment:
```suggestion
      p quakeLog_firstLine
```
Não precisa usar o keyword return. No ruby, a ultima linha de uma função é sempre retornada. Além disso, se quiser retornar o valor para testar ele no RSpec ao invés de pegar do `STDOUT` pode trocar o `puts` pelo `p`. O `p` vai jogar o valor no STDOUT e vai retornar o valor printado.